packages/web/src/components/nav/desktop/AccountDetails.tsx

```diff
@@ -1,66 +1,130 @@
 
 import { AvatarLegacy } from 'components/avatar/AvatarLegacy'
 import { TextLink, UserLink } from 'components/link'
 import { useFlag } from 'hooks/useRemoteConfig'
 import { useSelector } from 'utils/reducer'
 import { SIGN_IN_PAGE, profilePage } from 'utils/route'
 
-import styles from './AccountDetails.module.css'
 import { AccountSwitcher } from './AccountSwitcher/AccountSwitcher'
 
 const { getAccountUser } = accountSelectors
 
 const messages = {
   haveAccount: 'Have an Account?',
+  managedAccount: 'Managed Account',
   signIn: 'Sign in'
 }
 
 export const AccountDetails = () => {
   const account = useSelector((state) => getAccountUser(state))
-  const { isEnabled: isManagerModeEnabled } = useFlag(FeatureFlags.MANAGER_MODE)
+  const { color } = useTheme()
+  const { isEnabled: isManagerModeEnabled = false } = useFlag(
+    FeatureFlags.MANAGER_MODE
+  )
+  const isManagedAccount = useIsManagedAccount() && isManagerModeEnabled
 
   const profileLink = profilePage(account?.handle ?? '')
 
   return (
-    <div className={styles.userHeader}>
-      <div className={styles.accountWrapper}>
-        <AvatarLegacy userId={account?.user_id} />
-        <div className={styles.userInfo}>
-          {account ? (
-            <>
-              <UserLink
-                textVariant='title'
-                size='s'
-                strength='weak'
-                userId={account.user_id}
-                badgeSize='xs'
-              />
-              <TextLink
-                size='xs'
-                to={profileLink}
-              >{`@${account.handle}`}</TextLink>
-            </>
-          ) : (
-            <>
-              <Text variant='body' size='s' strength='strong'>
-                {messages.haveAccount}
-              </Text>
-              <TextLink
-                to={SIGN_IN_PAGE}
-                variant='visible'
-                size='xs'
-                strength='weak'
-              >
-                {messages.signIn}
-              </TextLink>
-            </>
-          )}
-        </div>
-        {isManagerModeEnabled && account ? <AccountSwitcher /> : null}
-      </div>
-    </div>
+    <Flex direction='column' pb='unit5' w='100%'>
+      {isManagedAccount ? (
+        <Box pv='xs' ph='m' backgroundColor='accent'>
+          <Text variant='label' size='xs' color='staticWhite'>
+            {messages.managedAccount}
+          </Text>
+        </Box>
+      ) : null}
+      <Flex
+        pt='l'
+        pr='s'
+        pb='s'
+        pl='m'
+        css={{
+          ...(isManagedAccount
+            ? {
+                '&:before': {
+                  content: '""',
+                  position: 'absolute',
+                  top: 0,
+                  left: 0,
+                  height: '100%',
+                  width: '100%',
+                  backgroundColor: color.background.accent,
+                  opacity: 0.03
+                },
+                borderBottom: `1px solid ${color.border.strong}`
```

Comment:
borderBottom='strong' prop should work

Reply:
Oh good call. Will switch. Not sure why I did it this way other than keeping all the manager-mode and "is this a managed account" styling logic in one place.